sources/engine/Xenko.Engine/Updater/UpdateEngine.cs

```diff
@@ -738,6 +738,27 @@ public override string ToString()
             {
                 return $"{Owner.Name}.{Name}";
             }
+
+            public bool Equals(UpdateKey other)
+            {
+                return Owner == other.Owner && Name == other.Name;
+            }
+
+            public override bool Equals(object obj)
+            {
+                if (ReferenceEquals(null, obj)) return false;
+                return obj is UpdateKey && Equals((UpdateKey)obj);
```

Comment:
Can be simplified to ``return obj is UpdateKey key && Equals(key);`` the null test above isn't necessary as the is test already returns false for nulls.

Reply:
Ok, I copied it from an existing code. I assume this means we can use C# 7 features (I think I saw some in use already). Will make this change.

Reply:
Please advise if you'd prefer me to amend the existing commit and force push instead of making another commit.